_sphinx/conf.py

```diff
@@ -45,6 +45,9 @@
 # The full version, including alpha/beta/rc tags.
 release = u''
 
+# -- Define the default role to use for links ----------------------------
```

Comment:
@apojomovsky nit: can we take those dashes out so that it looks like the rest?

Reply:
sure, done!